applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -1203,13 +1212,18 @@ int le_audio_send(uint8_t const *const data, size_t size)
 		audio_datapath_just_in_time_check_and_adjust(tx_info.ts);
 	}
 
-	ret = iso_stream_send(data, sdu_size, headsets[AUDIO_CH_L]);
+	ret = iso_stream_send(enc_audio.data, data_size_pr_stream, headsets[AUDIO_CH_L]);
 	if (ret) {
 		LOG_DBG("Failed to send data to left channel");
 	}
 
 #if !CONFIG_STREAM_BIDIRECTIONAL
-	ret = iso_stream_send(&data[sdu_size], sdu_size, headsets[AUDIO_CH_R]);
+	if (enc_audio.num_ch == 1) {
+		ret = iso_stream_send(enc_audio.data, data_size_pr_stream, headsets[AUDIO_CH_R]);
+	} else {
+		ret = iso_stream_send(&enc_audio.data[data_size_pr_stream], data_size_pr_stream,
```

Comment:
Also here we assume that CIS gateway has either 1 or 2 streams?
Should we make this more flexible?

Reply:
Changed